Install_or_Update.bat

```diff
@@ -1,8 +1,10 @@
-set PATH=%PATH%;%USERPROFILE%\anaconda3\Scripts;
-call activate.bat
-call conda remove --name geoapps --all
-call conda env update --file environment.yml  --prune
-call activate.bat geoapps
-call conda env config vars set KMP_WARNINGS=0
+set CONDA_EXE=conda
+:: if Conda executable is not in the PATH, uncomment and set the executable location below
+::set CONDA_EXE=%USERPROFILE%\AppData\Local\Continuum\anaconda3\Library\bin\conda.bat
+
```

Comment:
Looks like you need to call the activate.bat. I m getting this error otherwise :
![image](https://user-images.githubusercontent.com/55204635/138139352-ca463cc3-f98f-4e46-a457-a24637f4b6bc.png)

Reply:
add `call %CONDA_EXE% activate`

Reply:
not sure what happened in you env, but this is supposed to call conda.bat, never conda.exe
![image](https://user-images.githubusercontent.com/24550538/138344936-02c56e37-3477-4300-a682-12fc4a8d1546.png)

Might be fixed with the script I have introduced to find Conda.
Passing the env name to conda conf with `-n` or `--name` should work: `conda env config vars set my_var=value -n test-env` (from [Conda doc](
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#setting-environment-variables))

Reply:
ok sounds good, I ll test

Reply:
also I have moved this variable definition directly in the environment .yml file, so this command line is not necessary anymore